### DISCUSSION TITLE
issue with haxe

### DISCUSSION THREAD

**Discussion by moha0100110** [colyseus/colyseus#735] [General]
i got that error on all platforms except for html
Chunked encoding is not supported and the content-length header is required.
pls can some1 tell me how to fix it?

**Comment by endel** [comment]
On version 0.17 we are now preventing "chunked" encoding: https://github.com/colyseus/colyseus/blob/6bc066b6cc3551386e8d4df14f9f153fb4c35058/packages/core/src/router/default_routes.ts#L45-L49

The tink_http project does not support "chunked" encoding unfortunately.  https://github.com/haxetink/tink_http/issues/27
